core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitor.scala

```diff
@@ -405,46 +362,24 @@ class AvroUpsertMonitorActor(val pollInterval: FiniteDuration,
 
   }
 
-  private def lookupWorkspace(attributes: AvroUpsertAttributes): Future[Workspace] =
+  private def lookupWorkspace(attributes: CwdsUpsertAttributes): Future[Workspace] =
     dataSource.inTransaction { dataAccess =>
-      // db query: the SQL to find the workspace in the db
-      // errorId: if the workspace is not found, what identifier do we put in the error message?
-      val (dbQuery, errorId) = attributes match {
-        case cwdsAttributes: CwdsUpsertAttributes =>
-          (dataAccess.workspaceQuery.findById(cwdsAttributes.workspaceId.toString), cwdsAttributes.workspaceId.toString)
-        case importServiceAttributes: ImportServiceUpsertAttributes =>
-          (dataAccess.workspaceQuery.findByName(importServiceAttributes.workspace),
-           importServiceAttributes.workspace.toString
-          )
-      }
-      dbQuery.map {
+      dataAccess.workspaceQuery.findById(attributes.workspaceId.toString).map {
         case Some(workspace) => workspace
         case None =>
           throw new RawlsException(
-            s"Workspace $errorId not found while importing entities"
+            s"Workspace $attributes.workspaceId.toString not found while importing entities"
```

Comment:
Pretty sure this is going to output
```
"Workspace " + attributes + ".workspaceId.toString not found …"
```
suggest changing this to include braces (and remove the redundant toString):
```scala
s"Workspace ${attributes.workspaceId} not found …"
```

Reply:
Thanks for noticing this!